activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -72,10 +72,17 @@ def not(opts, *rest)
       #    # INNER JOIN "authors" ON "authors"."id" = "posts"."author_id"
       #    # INNER JOIN "comments" ON "comments"."post_id" = "posts"."id"
       #    # WHERE "authors"."id" IS NOT NULL AND "comments"."id" IS NOT NULL
+      #
+      # Additionally, you can define joine type in the scope and associated will not use default that is joins
```

Comment:
```suggestion
      # Additionally, you can define join type in the scope and +associated+ will not use default that it is `JOIN`.
```

Reply:
We also need to change the `Additionally` in the topic above, probably removing it.